gsk4/Gir.toml

```diff
@@ -90,6 +90,10 @@ cfg_condition = "feature = \"broadway\""
 name = "Gsk.GLShader"
 status = "generate"
 generate_builder = true
+    [[object.function]]
+    name = "get_arg_bool"
```

Comment:
Why this one needs an explicit rename but not the other ones such as arg_int?

Reply:
What I meant is that the return value is not an answer to "is this arg a boolean?", but the actual value of the `arg`.

Reply:
Ah I see. Makes sense.